### PR TITLE
updated indicator images to use css images instead of static images agai...

### PR DESCRIPTION
**Pull request by JayUnt**
...n

### CODE REVIEW

**Comment by JayUnt** [comment]
just an fyi, i did the ibu, abv and keg level since they all were back at the original static image format.

File: style.css [hunk 1]
```diff
@@ -175,7 +175,7 @@ span.tapcircle {
 	background-color: rgba(153,153,153,.3);
 }
 
-.BeerWrapper div{
+.BeerWrapper > div{
 	min-height:50px;
 	height:auto !important;
 	height:50px;
```
Comment (thadius856) [inline style.css:181]: Why not:

```
.BeerWrapper {
    height: auto;
    min-height:50px;
}
.BeerWrapper > div {
    height:50px;
}
```

?

**Comment by thadius856** [comment]
Looks like the BeerWrapper !important sneaked back in. Oh well. Committing.

**Comment by JayUnt** [comment]
I didnt want to make too many changes to the css because the last time i did, i lost all of my code because of bad merges.  I figured the style was where you guys wanted it since i just did a fork for the master right before making my changes and then committing.

**Comment by thadius856** [comment]
Understood. Will post a note on HBT thread about how to use pulls/forks/branches/commits, etc. momentarily.

**Pull request merged**